js/main.js

```diff
@@ -1,95 +1,70 @@
+	const pageArticle = document.querySelector('.page-article');
+	if (!pageArticle) {
+		return;
+	}
 
 	// Show author affiliation under authors list (for large screen only)
-	var authorString = $('.author-string-href');
-	$(authorString).click(function(event) {
-		event.preventDefault();
-		var elementId = $(this).attr('href').replace('#', '');
-		$('.article-details-author').each(function () {
-
-			// Show only targeted author's affiliation on click
-			if ($(this).attr('id') === elementId && $(this).hasClass('hideAuthor')) {
-				$(this).removeClass('hideAuthor');
-			} else {
-				$(this).addClass('hideAuthor');
-			}
-		});
-		// Add specifiers to the clicked author's link
-		$(authorString).each(function () {
-			if ($(this).attr('href') === ('#' + elementId) && !$(this).hasClass('active')){
-				$(this).addClass('active');
-				$(this).children('.author-plus').addClass('hide');
-				$(this).children('.author-minus').removeClass('hide');
-			} else if ($(this).attr('href') !== ('#' + elementId) || $(this).hasClass('active')) {
-				$(this).removeClass('active');
-				$(this).children('.author-plus').removeClass('hide');
-				$(this).children('.author-minus').addClass('hide');
-			}
-		});
-	})
-})(jQuery);
-
-
-// initiating tag-it
-$(document).ready(function() {
-	$('#tagitInput').each(function() {
-		var autocomplete_url = $(this).data('autocomplete-url');
-		$(this).tagit({
-			fieldName: $(this).data('field-name'),
-			allowSpaces: false,
-			autocomplete: {
-				source: function(request, response) {
-					$.ajax({
-						url: autocomplete_url,
-						data: {term: request.term},
-						dataType: 'json',
-						success: function(jsonData) {
-							if (jsonData.status == true) {
-								response(jsonData.content);
-							}
-						}
-					});
-				},
-			},
+	const authorLinks = document.querySelectorAll('.author-string-href');
+	authorLinks.forEach(function (authorLink) {
+		authorLink.addEventListener('click', function (event) {
+			event.preventDefault();
+			const elementId = this.getAttribute('href').replace('#', '');
+			document.querySelectorAll('.article-details-author').forEach((details) => {
+
+				// Show only targeted author's affiliation on click
+				if (details.getAttribute('id') === elementId && details.classList.contains('hideAuthor')) {
+					details.classList.remove('hideAuthor');
+				} else {
+					details.classList.add('hideAuthor');
+				}
+			});
+
+			// Add specifiers to the clicked author's link
+			authorLinks.forEach((sibling) => {
+				if (authorLink === sibling && !sibling.classList.contains('active')) {
+					sibling.classList.add('active');
+					sibling.querySelector(':scope .author-plus').classList.add('hidden');
+					sibling.querySelector(':scope .author-minus').classList.remove('hidden');
+				} else {
+					sibling.classList.remove('active');
+					sibling.querySelector(':scope .author-plus').classList.remove('hidden');
+					sibling.querySelector(':scope .author-minus').classList.add('hidden');
```

Comment:
I _think_ `:scope` is not necessary here. When `.querySelector()` is called from an `Element`, it only searches in child elements.

Reply:
Yep, looks like I confused it with `querySelectorAll()`